frontend/src/reusable/default/DefaultPromptDefinitions.ts

```diff
@@ -5,58 +5,60 @@ import SliderPrompt from "../inputs/SliderPrompt";
 import TextPrompt from "../inputs/TextPrompt";
 
 export const defaultPromptDefinitions: PromptDefinition[] = [
```

Comment:
it would be cool to pull these definitions from the backend automatically so we don't have to "match it up" on the frontend, and it just knows stuff like keys and ranges from a single point of modification. Pain in the ass tho, so save it for next sprint